src/ray/gcs/gcs_server/gcs_node_manager.h

```diff
@@ -167,6 +173,8 @@ class GcsNodeManager : public rpc::NodeInfoHandler {
   std::shared_ptr<gcs::GcsTableStorage> gcs_table_storage_;
   /// Raylet client pool.
   std::shared_ptr<rpc::NodeManagerClientPool> raylet_client_pool_;
+  /// Cluster ID.
+  const ClusterID &cluster_id_;
```

Comment:
I think it's safer to remove `&` to avoid potential dangling pointer. Copying ClusterID is cheap anyway.

Reply:
It won't be dangling because gcs_node_manager lives in gcs_server